webapp/src/shared/components/Avatar/Avatar.tsx

```diff
@@ -4,7 +4,7 @@ import { Icon, IconVariant, IconSize } from '../index';
 import { StatusVariant } from '../Status/Status';
 import { Color } from '../../types';
 
-type AvatarProps = {
+export type AvatarProps = {
```

Comment:
Why do you need to export this? the url is just a string and the status is a `StatusVariant` which is a string type limited to a set of values. This is, you don't need to import this to pass the value of the stauts, you just need to pass one of allowed values of it. This is indeed one of the benefits of creating types like that, you don't have to import the type itself because it's just a string